### ISSUE TITLE
Removing \maketitle results in error

### ISSUE THREAD

**Issue by elbotho** [Lodifice/mfnf-pdf-export#105]
```
! \textfont6 is undefined (character 50).
l.294 At vero $x^{2}$
                    eos et accusamus et iusto $\exp(x)$ dignissimos ducimus ...
```

**Comment by kulla** [comment]
Possible a solution: https://www.tug.org/pipermail/tex-live/2013-July/033904.html

**Comment by Lodifice** [comment]
Fixed in 856c5fa1e743042fdb1a1a62fedf650c6593803c.

This was not totally random. Apparently, if both `tocstyle` and `fontspec` are used and no `\maketitle` is done, some math fonts not there in the size TeX thinks they should have.
